README.md

```diff
@@ -59,13 +59,14 @@ List of targeted installs to run on hosts. Available options are:
 - `apm-nodejs` (Linux)
 - `apm-dotnet` (Linux & Windows)
 - `mysql` (Linux)
+- `nginx` (Linux)
 
 Important Notes:
 
 - the `logs` target requires `infrastructure`, and an error will be thrown if `logs` is specified without `infrastructure`.
 - the `apm-nodejs` agent installation is supported only for apps managed by [PM2](https://pm2.keymetrics.io/). To install the agent using a package manager such as `npm` or `yarn` or via other installation paths, please reference our [docs](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent/).
 - the `apm-dotnet` agent installation for Windows is supported only for apps hosted by [IIS](https://www.iis.net/). Linux installations are only supported for .NET applications which run as a `systemd` service.
-- the `mysql` targets requires and installs `infrastructure`.
+- the `mysql` and `nginx` targets requires and installs `infrastructure`.
```

Comment:
Now that we have two, maybe we want something like a list for future OHI targets:

```
- the following integrations require the infrastructure agent to be installed:
  - mysql
  - nginx
```